packages/jsdoc-plugins/lib/validator/types.js

```diff
@@ -35,6 +35,7 @@ const BASIC_TYPES = [
 	'ClientRect',
 
 	// Web APIs
```

Comment:
Hmm, but that's something more than just DOM. I should create another category or write it at the top level.